.buildkite/verify-yaml.sh

```diff
@@ -5,7 +5,7 @@ cd "$(dirname "${BASH_SOURCE[0]}")/.."
 
 gcloud container clusters get-credentials cloud --zone us-central1-f --project sourcegraph-dev
 
-kubectl apply --dry-run --validate --recursive -f base/ --context=gke_sourcegraph-dev_us-central1-f_cloud
-kubectl apply --dry-run --validate --recursive -f configure/ --context=gke_sourcegraph-dev_us-central1-f_cloud
+kustomize build ./base/. | kubectl apply --dry-run --validate --context=gke_sourcegraph-dev_us-central1-f_cloud -f -
+kustomize build ./configure/. | kubectl apply --prune -l deploy=sourcegraph --dry-run --validate --context=gke_sourcegraph-dev_us-central1-f_cloud -f -
```

Comment:
`./configure/.` doesn't contain any `kustomize.yaml` files which is why this line is failing on the pipeline

Reply:
Hey will, thanks for looking into this! I will fix the rest of the errors about the kustomize files tomorrow. 

As for the gloud auth plugin error i mentioned in the original thread, you can find it in the commit I made before today. (So sorry for the confusions cause since I haven't checked the build after the commits I made today lol)